modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/ReliableCatalogVersions.java

```diff
@@ -38,6 +38,8 @@ public ReliableCatalogVersions(SchemaSyncService schemaSyncService, CatalogServi
     /**
      * Returns Catalog version corresponding to the given timestamp.
      *
+     * <p>This should only be used when the startup procedure is complete.
```

Comment:
I think it would also be beneficial specify "why"  it can only be used after the startup.